src/wooting-usb.h

```diff
@@ -17,16 +17,32 @@ typedef enum RGB_PARTS {
 	PART4
 } RGB_PARTS;
 
+
+typedef enum WOOTING_DEVICE_TYPE {
+	// 10 Keyless Keyboard. E.g. Wooting One
+	DEVICE_KEYBOARD_TKL = 1,
+	
+	// Full Size keyboard. E.g. Wooting Two
+	DEVICE_KEYBOARD = 2
+} WOOTING_DEVICE_TYPE;
+
 typedef struct WOOTING_USB_META {
 	bool connected;
-	bool wooting_one;
-	bool wooting_two;
-} WOOTING_USB_META;
+	const char* model;
```

Comment:
Will the strings go through fine when the DLL gets called from different languages? I usually define sizes of strings in C, because in embedded dynamic memory can be tricky. If this will work fine it's OK, I'm just a little paranoid when it comes to C strings 😛

Reply:
AFAIK they should be fine. I was testing with my C# wrapper and it didn't have any issues with the string. I believe that any proper ffi interface should be designed to be capable of handling null-terminated strings but I'll double-check that its support is ubiquitous